packages/editor/src/browser/editor-command.ts

```diff
@@ -59,6 +63,11 @@ export namespace EditorCommands {
         category: EDITOR_CATEGORY,
         label: 'Change Language Mode'
     };
+    export const CHANGE_ENCODING: Command = {
+        id: 'textEditor.change.encoding',
+        category: EDITOR_CATEGORY,
+        label: 'Change FILE Encoding'
```

Comment:
For consistency, perhaps rename this to **`Change File Encoding`**.

Reply:
Yes, you're right.

Reply:
@vince-fugnitto Hi, I've just fixed the consistency problem.